doc/api/http.md

```diff
@@ -1354,16 +1354,47 @@ added: v0.3.6
 -->
 
 Since most requests are GET requests without bodies, Node.js provides this
-convenience method. The only difference between this method and [`http.request()`][]
-is that it sets the method to GET and calls `req.end()` automatically.
+convenience method. The only difference between this method and
+[`http.request()`][] is that it sets the method to GET and calls `req.end()`
+automatically. Note that response data must be consumed in the callback
+for reasons stated in [`http.ClientRequest`][] section.
 
-Example:
+The `callback` is invoked with a single argument that is an instance of
+[`http.IncomingMessage`][]
+
+JSON Fetching Example:
 
 ```js
-http.get('http://www.google.com/index.html', (res) => {
-  console.log(`Got response: ${res.statusCode}`);
-  // consume response body
-  res.resume();
+http.get('http://jsonplaceholder.typicode.com/posts/1', (res) => {
```

Comment:
I wonder if we should just use a fake URL, like `'http://example.org/posts/1.json'` or something? That way we have more consistency/reliability (e.g. example.org is always a reserved domain name and will never go away).